winlogbeat/eventlog/wineventlog.go

```diff
@@ -144,48 +145,36 @@ func (l *winEventLog) Name() string {
 }
 
 func (l *winEventLog) Open(state checkpoint.EventLogState) error {
-	var bookmark win.EvtHandle
-	var err error
-	if len(state.Bookmark) > 0 {
-		bookmark, err = win.CreateBookmarkFromXML(state.Bookmark)
-	} else if state.RecordNumber > 0 {
-		bookmark, err = win.CreateBookmarkFromRecordID(l.channelName, state.RecordNumber)
-	}
-	if err != nil {
-		return err
-	}
-	defer win.Close(bookmark)
-
 	if l.file {
-		return l.openFile(state, bookmark)
+		return l.openFile(state)
 	}
-	return l.openChannel(bookmark)
+	return l.OpenChannel()
```

Comment:
Is it right, I see that this method does not exported?

Reply:
No its not right. seems it was missed to update. Will change it.